Mathlib/Data/Multiset/LocallyFinite.lean

```diff
@@ -100,19 +100,20 @@ theorem Ioo_eq_zero_of_le (h : b ≤ a) : Ioo a b = 0 :=
 
 variable (a)
 
-@[simp]
+-- Porting note: simp can prove this -- @[simp]
 theorem Ico_self : Ico a a = 0 := by rw [Ico, Finset.Ico_self, Finset.empty_val]
 #align multiset.Ico_self Multiset.Ico_self
 
-@[simp]
+-- Porting note: simp can prove this -- @[simp]
 theorem Ioc_self : Ioc a a = 0 := by rw [Ioc, Finset.Ioc_self, Finset.empty_val]
 #align multiset.Ioc_self Multiset.Ioc_self
 
-@[simp]
+-- Porting note: simp can prove this -- @[simp]
 theorem Ioo_self : Ioo a a = 0 := by rw [Ioo, Finset.Ioo_self, Finset.empty_val]
 #align multiset.Ioo_self Multiset.Ioo_self
 
-variable {a b c}
+-- Porting note: commented out as not needed.
+-- variable {a b c}
```

Comment:
```suggestion
variable {a}
```
My impression is that Lean would complain about `b` and `c`. But `a` should be made implicit, because it was made explicit on line 101.